code/game/machinery/fax.dm

```diff
@@ -16,6 +16,7 @@
 	var/sendcooldown = 0
 
 	var/department = "Corporate Liasion"
+	var/selected = "Nanotrasen"
```

Comment:
Make the name nanotrasen a define to stop misspellings or is that too far?

Reply:
The user can only pick from a list so it cannot be misspelled